.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: CDK Deployment
+
+on:
+  push:
+    branches:
+      - main
+      - stg
+      - dev
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
**Suggestion:** Add 'fail-fast: false' to the 'jobs.deploy.strategy' to prevent cancelling all runs if one fails, which is useful for parallel deployments.
```suggestion
jobs:
  deploy:
    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
```